README.md

```diff
@@ -1,38 +1,39 @@
 # proposals
```

Comment:
Just noticed this: Shouldn't this be capitalized?

Reply:
not really as repo name is proposals, but not a big deal - will captilize.